File: languages/python/hello.py
import sys


# main function/procedure
def main ():
    print ("Hello, World\n")    # print hello world to the stdout device
    return 0                    # exit with error code 0 (success)


# program main entry point
if __name__ == "__main__":      # check if the program was run directly
    sys.exit (main ())          # call main, exit with errorcode as ret


# end-of-file

File: languages/c/hello.c
#include <stdio.h>
#include <stdlib.h>


/* main entry point */
int
main (void)
{
	printf ("Hello, World!\n");	/* print hello world to the stdout device */
	fflush (stdout);		/* flush the stdout device, required on some systems */

	return EXIT_SUCCESS;		/* return a success */
}


/* end-of-file */
